pom.xml

```diff
@@ -91,20 +91,20 @@
         <jackson.databind.version>2.14.1</jackson.databind.version>
         <jvmbrotli.version>0.2.0</jvmbrotli.version>
         <logback.version>1.4.5</logback.version>
-        <swagger.ui.version>4.10.3</swagger.ui.version> <!-- used for testing webjars -->
         <slf4j.version>2.0.6</slf4j.version>
         <gson.version>2.10.1</gson.version>
+        <okhttp.version>4.10.0</okhttp.version>
```

Comment:
We use this library not only in tests, but also testtools module, so I think it should be here as it's not a test-scope only dependency.

Reply:
Agreed!